### DISCUSSION TITLE
recipes/breakfast/veggie-chickpea-hash/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#871] [Comments]
# recipes/breakfast/veggie-chickpea-hash/

A collection of my recipes.

https://nicholaswilde.io/recipes/breakfast/veggie-chickpea-hash/

**Comment by nicholaswilde** [comment]
Add a splash of lemon juice before adding the egg.
Cover the dish to cook the top of the egg.
